src/services/ai-server/ai-persona-service/dto/ai.persona.service.invocation.dto.input.ts

```
@@ -0,0 +1,126 @@
+    nullable: true,
+    description: 'The context needed for the result handler',
+  })
+  roomDetails?: RoomDetails;
+
+  @ValidateIf(handler => handler.action === InvocationResultAction.POST_REPLY)
+  @IsNotEmpty({
+    message:
+      'roomDetails with roomID, threadID and communicationID is required when action is POST_REPLY',
+  })
+  validateRoomDetails() {
+    return this.roomDetails && this.roomDetails.threadID;
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)